cli/workers.rs

```diff
@@ -70,6 +84,7 @@ pub fn spawn<B: WorkerBehavior + 'static>(
   let _tid = builder
     .spawn(move || {
```

Comment:
I've been meaning to change `Worker::spawn()` so that it no longer creates a new thread. The reason is that Isolates (and Workers) implement the Future trait, so they can be scheduled inside the Tokio thread pool. Because they implement Future, they don't even have a dedicated thread. So it seems like we shouldn't be spawning a special thread for them.
What do you think?

Reply:
I think the compiler worker will still need it's own thread until we convert all the execute functions to async, but I can do this for other workers.